tooling/src/publish.js

```diff
@@ -28,46 +28,37 @@ function versionAndPublish() {
     process.exit(1);
   }
   const contactCenterPath = './packages/contact-center';
-  const dependencies = ['@webex/cc-store'];
 
   try {
-    const ccFolder = fs
+    const workspaceData = fs
       .readdirSync(contactCenterPath, {withFileTypes: true})
-      .filter((dirent) => {
-        return dirent.isDirectory();
-      })
+      .filter((dirent) => dirent.isDirectory())
       .map((dirent) => {
-        try {
-          const packageJsonPath = path.join(contactCenterPath, dirent.name, 'package.json');
-          const packageData = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
+        const packageJsonPath = path.join(contactCenterPath, dirent.name, 'package.json');
+        const packageData = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
 
-          console.log(`Removing stable version from package.json for ${dirent.name}`);
-          removeStableVersion(packageJsonPath, packageData);
-          return packageData.name;
-        } catch (error) {
-          throw new Error(`Error reading package.json in ${dirent.name}`, error);
-        }
+        console.log(`Removing stable version from package.json for ${dirent.name}`);
+        removeStableVersion(packageJsonPath, packageData); // Assuming this function is defined elsewhere
+        return packageData.name;
       });
-    // Separate dependency workspaces and other workspaces
-    const dependencyWorkspaces = ccFolder.filter((fileName) => dependencies.includes(fileName));
 
-    const otherWorkspaces = ccFolder.filter((fileName) => !dependencies.includes(fileName));
-
-    const publishWorkspace = (workspaceName) => {
-      console.log(`Publishing new version for ${workspaceName}: ${newVersion}`);
-
-      // Update version in the workspace
-      execSync(`yarn workspace ${workspaceName} version ${newVersion}`, {stdio: 'inherit'});
+    // Update version in the workspace
+    const updateVersions = (workspace) => {
+      console.log(`Publishing new version for ${workspace}: ${newVersion}`);
+      execSync(`yarn workspace ${workspace} version ${newVersion}`, {stdio: 'inherit'});
+    };
 
-      // Publish the package
-      execSync(`yarn workspace ${workspaceName} npm publish --tag ${branchName}`, {stdio: 'inherit'});
+    // Publish the package
+    const publishWorkspace = (workspace) => {
+      console.log(`Updating version for ${workspace}: ${newVersion}`);
+      execSync(`yarn workspace ${workspace} npm publish --tag ${branchName}`, {stdio: 'inherit'});
```

Comment:
- Both the version and publish are already inside a try catch adding more try catch might be an overkill.
- The new version that we are getting is from semantic-release so we can trust that it will be in the correct format